### DISCUSSION TITLE
Netlify-плагин для a11y

### DISCUSSION THREAD

**Discussion by furtivite** [doka-guide/platform#33] [Q&A] [answered]
У Netlify есть плагин, который крашит сборку, если сайт недостаточно отвечает параметрам доступности. Например, у картинок нет alt-описания. 

[Вот документация](https://github.com/netlify-labs/netlify-plugin-a11y#readme). 

Хотим?

**Comment by pepelsbey** [comment, marked as answer]
Я бы шёл по другому пути с такими фичами. Не по такому:

1. Есть клёвый плагин
2. Давайте прикрутим
3. Кстати, он решает такую задачу

А по такому:

1. У нас недоступный сайт
2. Есть вот такой инструмент проверки
3. Давайте добавим автопроверку?

То есть от задачи, а не от инструмента :) Плюс валить сборку — это плохая идея, нужно валить тесты в пулреквестах.

Я бы лучше прикрутил к платформе валидатор W3C.

**Reply by pepelsbey** [reply]
Добавил задачку про валидатор #122
